src/treemap/inherent.rs

```diff
@@ -113,14 +113,14 @@ impl RoaringTreemap {
     where
         R: RangeBounds<u64>,
     {
-        let (range, is_empty) = util::convert_range_to_inclusive(range);
-        if is_empty {
+        let (start, end);
+        if let Some(range) = util::convert_range_to_inclusive(range) {
+            start = *range.start();
+            end = *range.end();
+        } else {
             return 0;
         }
```

Comment:
I would prefer that we use a match here instead, it is easier to read and more rust idiomatic :)
Can you do this at the other places in your changes, please?

```suggestion
        let (start, end) = match util::convert_range_to_inclusive(range) {
            Some(range) => (*range.start(), *range.end()),
            None => return 0,
        };
```